### DISCUSSION TITLE
Reload the directory

### DISCUSSION THREAD

**Discussion by ghiesz** [iterate-ch/cyberduck#15155] [Q&A] [answered]
In the manual is a point about an reload over the context menu, but i do not find this there.
Working on Ventura.
How can i update an file ore directory via relaod.
By now i have to quit and restart mountain duck, sometimes two times to get actual files.
Thanks for help

**Comment by tatze96** [comment, marked as answer]
Mountain Duck periodically polls for changes for currently open folders in Finder, regardless of the used connection mode. Additionally, you can refresh the file listing manually with the help of the _Reload_ option in the context menu.

To see the mentioned context menu option, you'll have to activate the Finder Extension in `System Settings → Privacy & Security → Extensions → Added Extensions`. After activating the Finder Extension, you can refresh the file listing by right-clicking on white space within the desired folder and choosing `Mountain Duck → Reload` within the appearing context menu.

For more information regarding the reload functionality, refer to our [wiki page](https://docs.cyberduck.io/mountainduck/interface/#reload).
